src/renderer/entities/staking/api/validatorsService.ts

```diff
@@ -222,22 +243,24 @@ async function getParentIdentities(
     if (!identity || identity.isNone) return acc;
 
     const { parent, sub, subName } = subIdentities[index];
-    const { info } = identity.unwrap(); // { judgements, info }
-    const { display, web, riot, email, twitter } = info;
+    const unwrappedIdentity = identity.unwrap();
+    // HINT: in runtime 1_4_0 unwrappedIdentity returns Option<(identity, rest)>
+    const data = 'info' in unwrappedIdentity ? unwrappedIdentity : unwrappedIdentity[0];
+    const { display, web, email, twitter } = data.info; // { data also includes: judgements, deposit }
 
-    const payload: Identity = {
+    acc[sub] = {
       subName,
       email: email.isRaw ? u8aToString(email.asRaw) : email.value.toString(),
-      riot: riot.isRaw ? u8aToString(riot.asRaw) : riot.value.toString(),
       twitter: twitter.isRaw ? u8aToString(twitter.asRaw) : twitter.value.toString(),
       website: web.isRaw ? u8aToString(web.asRaw) : web.value.toString(),
+      // riot: riot.isRaw ? u8aToString(riot.asRaw) : riot.value.toString(),
```

Comment:
what reason to comment this code and not to delete ?

Reply:
Hm, I agree, we can safely remove it, because we don't show riot/matrix anyway

Reply:
why not show it though (I know it's not here)? I find it useful at times for actual real time messaging, rather than the other platforms.

Reply:
> why not keep it, and actually show it? (I know it's not here, could be in another PR) I find it useful at times for actual real time messaging, rather than the other platforms.

In current design we don't show `riot/matrix` and don't have any plans to do so